ThingIFSDK/ThingIFSDK/ThingIFAPI.swift

```diff
@@ -780,6 +780,10 @@ open class ThingIFAPI: Equatable, NSCoding {
 
     /** Aggregate history states
 
+     `AggregatedValueType` represents calcuated value with
+     `Aggregation.Functions`. The value is numerical so developers use
+     `Int` or `Double` and so on as `AggregatedValueType`.
+
```

Comment:
I think the description can be improved like:

`AggregatedValueType` represents  type of calculated value with `Aggregation.Functions`. 
If the function is `max`, `min`, `mean` or `sum`, the type must be same as type  of field.
If the function is `count`, the type must be Int.